src/FastBroadcast.jl

```diff
@@ -222,12 +222,12 @@ end
 
 function broadcast_expr!(ex)
     ex isa Expr || return ex
-    update = findfirst(isequal(ex.head), (:(+=), :(-=), :(*=), :(/=), :(\=), :(^=), :(&=), :(|=), :(⊻=), :(÷=)))
+    update = findfirst(isequal(ex.head), (:(+=), :(-=), :(*=), :(/=), :(\=), :(^=), :(&=), :(|=), :(⊻=), :(÷=), :(.+=), :(.-=), :(.*=), :(./=), :(.\=), :(.^=), :(.&=), :(.|=), :(.⊻=), :(.÷=)))
     if update ≢ nothing
-        lhs = Expr(:call, (:(+), :(-), :(*), :(/), :(\), :(^), :(&), :(|), :(⊻), :(÷))[update], ex.args[1], ex.args[2])
+        lhs = Expr(:call, (:(+), :(-), :(*), :(/), :(\), :(^), :(&), :(|), :(⊻), :(÷))[update % 10], ex.args[1], ex.args[2])
         ex = Expr(:(=), ex.args[1], lhs)
     end
-    if Meta.isexpr(ex, :(=), 2)
+    if Meta.isexpr(ex, :(=), 2) || Meta.isexpr(ex, :(.=), 2)
```

Comment:
Do we need fast broadcast for `.=`? I thought `copyto!` is often preferred.

Reply:
It will dispatch to `copyto!`.
OrdinaryDiffEq was doing some variant of `@.. x .= y` in a few places, which cased errors because this expand to `fast_broadcast!(x .= y)` without this PR.